debian/big-brother.bigbro.service

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Documentation=https://github.com/braintree/big_brother
+Description=IPVS weight coordinator.
+Before=multi-user.target
+Before=graphical.target
```

Comment:
@mrak why do you need these Before statements?

Reply:
That looks like a miss. I'll correct it